addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,159 @@
+			<channel id="rain-raw" typeId="rain-raw" />
+			<channel id="rain-currenthour" typeId="rain-currenthour" />
+			<channel id="rain-lasthour" typeId="rain-lasthour" />
+			<channel id="solar-power" typeId="solar-power" />
+			<channel id="signal-strength" typeId="system.signal-strength" />
+			<channel id="low-battery" typeId="system.low-battery" />
+		</channels>
+
+		<config-description>
+			<parameter name="channel" type="integer" required="true">
+				<label>Channel</label>
+				<description>Specifies the channel the sensor is using</description>
+				<options>
+					<option value="1">Channel 1</option>
+					<option value="2">Channel 2</option>
+					<option value="3">Channel 3</option>
+					<option value="4">Channel 4</option>
+					<option value="5">Channel 5</option>
+					<option value="6">Channel 6</option>
+					<option value="7">Channel 7</option>
+					<option value="8">Channel 8</option>
+				</options>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="indoor-temperature">
+		<item-type>Number</item-type>
+		<label>Indoor Temperature</label>
+		<description>Current temperature in degrees celsius</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f °C">
+		</state>
+	</channel-type>
+
+	<channel-type id="outdoor-temperature">
+		<item-type>Number</item-type>
+		<label>Outdoor Temperature</label>
+		<description>Current temperature in degrees celsius</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f °C">
+		</state>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Outdoor Humidity</label>
+		<description>Current humidity in %</description>
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%d %%">
+		</state>
+	</channel-type>
+
+	<channel-type id="pressure">
+		<item-type>Number</item-type>
+		<label>Pressure</label>
+		<description>Current pressure in millibar</description>
+		<category>Pressure</category>
+		<state readOnly="true" pattern="%.1f hPa">
+		</state>
+	</channel-type>
+
+	<channel-type id="wind-direction">
+		<item-type>Number</item-type>
+		<label>Wind Direction</label>
+		<description>Wind direction</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.0f °">
+		</state>
+	</channel-type>
+
+	<channel-type id="wind-speed">
+		<item-type>Number</item-type>
+		<label>Wind Speed</label>
+		<description>Wind speed</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.2f m/s">
+		</state>
+	</channel-type>
+
+	<channel-type id="rain-raw">
+		<item-type>Number</item-type>
+		<label>Raw rainfall count</label>
+		<description>A counter between 0 and 255 in 0.01mm steps</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.1f">
+		</state>
+	</channel-type>
+
+	<channel-type id="rain-lasthour">
+		<item-type>Number</item-type>
+		<label>Rainfall in previous hour</label>
```

Comment:
Ah, ok, just read the docs :-)
Wouldn't it make sense to mark at least one (if not both) of these channels as advanced?